### ISSUE TITLE
Could not run the ResNet-50 example of creating the standalone bundles

### ISSUE THREAD

**Issue by csbenk** [pytorch/glow#1604]
This is my log, after running `make run`:
```
File ?resnet50/predict_net.pbtxt? already there; not retrieving.

File ?resnet50/predict_net.pb? already there; not retrieving.

File ?resnet50/init_net.pb? already there; not retrieving.

image-classifier: for the -model_input_name option: must be specified at least once!
Makefile:43: recipe for target 'build/resnet50.o' failed
make: *** [build/resnet50.o] Error 1
```

After adding `-model_input_name=gpu_0/data ` to line 44 in the make file, I get 
```
File ?resnet50/predict_net.pbtxt? already there; not retrieving.

File ?resnet50/predict_net.pb? already there; not retrieving.

File ?resnet50/init_net.pb? already there; not retrieving.

Allocated weights of size: 102121984
Expected weights of size: 102121984
Loaded weights of size: 102121984 from the file resnet50.weights
Allocated mutable weight variables of size: 606144
Loaded image: /datadrive/bhaskar/renesasai/glow/tests/images/imagenet/cat_285.png
Loaded images size in bytes is: 602112
Copying image data into mutable weight vars: 602112 bytes
Expected to find variable 'output'
resnet50: resnet50.cpp:267: const SymbolTableEntry &getMutableWeightVar(const BundleConfig &, const char *): Assertion `mutableWeightVar && "Expected to find a mutable weight variable"' failed.
Aborted (core dumped)
Allocated weights of size: 102121984
Expected weights of size: 102121984
Loaded weights of size: 102121984 from the file resnet50.weights
Allocated mutable weight variables of size: 606144
Loaded image: /datadrive/bhaskar/renesasai/glow/tests/images/imagenet/dog_207.png
Loaded images size in bytes is: 602112
Copying image data into mutable weight vars: 602112 bytes
Expected to find variable 'output'
resnet50: resnet50.cpp:267: const SymbolTableEntry &getMutableWeightVar(const BundleConfig &, const char *): Assertion `mutableWeightVar && "Expected to find a mutable weight variable"' failed.
Aborted (core dumped)
Allocated weights of size: 102121984
Expected weights of size: 102121984
Loaded weights of size: 102121984 from the file resnet50.weights
Allocated mutable weight variables of size: 606144
Loaded image: /datadrive/bhaskar/renesasai/glow/tests/images/imagenet/zebra_340.png
Loaded images size in bytes is: 602112
Copying image data into mutable weight vars: 602112 bytes
Expected to find variable 'output'
resnet50: resnet50.cpp:267: const SymbolTableEntry &getMutableWeightVar(const BundleConfig &, const char *): Assertion `mutableWeightVar && "Expected to find a mutable weight variable"' failed.
Aborted (core dumped)
Makefile:22: recipe for target 'run' failed
make: *** [run] Error 134
```

Can someone point out if I have to make any more changes to get it running?

**Comment by nadavrot** [comment]
@csbenk Thank you for reporting the error. Is this a recent regression? Can you check it things worked a month ago?  FYI:  @opti-mix @qcolombet

**Comment by csbenk** [comment]
@nadavrot I started working just four days back.

**Comment by jfix71** [comment]
I put up a fix for the missing `-model_input_name` in https://github.com/pytorch/glow/pull/1606. This seems to not be a recent regression. I picked a random commit about a month ago to check, https://github.com/pytorch/glow/commit/3de6e36ca51390ee4dfcd355656196403221919c, and it was broken then too.

**Comment by qcolombet** [comment]
I'll have a look later today.

**Comment by qcolombet** [comment]
Don't know when it changed, but according to predict_net.pbtxt, the output of the network is "gpu_0/softmax", not output.
Checking.

**Comment by qcolombet** [comment]
Yep, that was the problem.

**Comment by qcolombet** [comment]
Should be fixed by #1608.
